lib/featuring.js

```diff
@@ -1,6 +1,6 @@
 class Featuring {
 	constructor(colaboradores) {
-		this.colaboradores	= colaboradores;
+		this.colaboradores = colaboradores;
```

Comment:
¿Te parece que un array de colaboradores necesite una clase aparte?

Reply:
Esto se hizo así ya que durante el objetivo 2, se llegó a la conclusión de que una colaboración era eso, un `array` de artistas. Se podría cambiar, por supuesto

Reply:
No te estoy preguntando eso.

Reply:
Lo veo útil ya que utilizo `feat instanceof Featuring` para discernir si se trata de un artista en solitario o de una colaboración.

De todas formas, investigarué sobre formas de poder identificar un `array`, ya que es verdad que no es necesario.

Reply:
Acabo de encontrar una forma que parece ser bastante correcta. Lo cambiaré entonces.

¡Gracias por su paciencia! :)

Reply:
> Lo veo útil ya que utilizo `feat instanceof Featuring` para discernir si se trata de un artista en solitario o de una colaboración.
> 
> De todas formas, investigarué sobre formas de poder identificar un `array`, ya que es verdad que no es necesario.

¿Como ver la longitud de un array?

Reply:
No, de saber si es un array o no, ya que se le podía pasar un `string` en vez de un objeto de esa clase en el constructor de `playlist`

Reply:
Hay también arrays de un elemento... De veras, no liéis las cosas más de lo estrictamente necesario.

Reply:
Claro, pero entonces ya no es una colaboración, por eso había esa distinción de casos.

Pese a esto, en cuanto a efectos de implementación es mucho más simple pasar arrays de un elemento y tomarlo todo como colaboraciones y el resultado sería el mismo.